lib/rules/no-self-assign.js

```diff
@@ -165,6 +165,8 @@ function eachSelfAssignment(left, right, props, report) {
 
 module.exports = {
     meta: {
+        type: "suggestion",
```

Comment:
This seems like it could be `problem` since it usually catches bugs, but I don't feel strongly about it.

Reply:
I agree.